### DISCUSSION TITLE
Further Changes

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/awesome-github-profiles#904] [General]
Please watch this video: 
https://www.loom.com/share/abf95955392349ba9e50d2cdb47b1d81?sid=529274e1-aa25-4d8e-abff-55b99b88a317

@HimangshuSharma01

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/495ea2f3-e4cf-45a2-a59d-002e8b707723)
add the linkedin and email Id so people can reach out to you. refer mne. 


change your "1st year all text to " 
Opensource developer • Maintainer @recodehive 

sponsor me $1 I will send you back so can earn badge in such a way

**Reply by HimangshuSharma01** [reply]
Sir it shows invalid card type when ever i put my details

**Reply by sanjay-kv** [reply]
ohk maybe not supported card

**Comment by HimangshuSharma01** [comment]
@sanjay-kv  Sir I'm in my college doing it as soon as possible

**Comment by HimangshuSharma01** [comment]
![image](https://github.com/user-attachments/assets/447ddd6d-ad37-4aa8-8132-7eb16fcacc4f)
@sanjay-kv Sir i'm not understanding this kindly help

**Comment by HimangshuSharma01** [comment]
@sanjay-kv  sir i almost visited each and every issue from page 2 to the last one and did whatever was asked just the page one is left i will finish that by tomorrow . thank you sir .

**Reply by sanjay-kv** [reply]
thanks bro @HimangshuSharma01

**Comment by HimangshuSharma01** [comment]
@sanjay-kv Sir can I merge the remaining pull request and also the preloader one looks good

**Reply by sanjay-kv** [reply]
you can do the code review, dont merge. 
so once you do code review it will help improve your contribution graph and I need to do less review.

**Reply by HimangshuSharma01** [reply]
Okay sir

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/49b5aad7-8c76-4320-8258-3804963994e5)

its improving

**Reply by HimangshuSharma01** [reply]
> ![image](https://private-user-images.githubusercontent.com/30715153/378203328-49b5aad7-8c76-4320-8258-3804963994e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0Mzk5NjAsIm5iZiI6MTcyOTQzOTY2MCwicGF0aCI6Ii8zMDcxNTE1My8zNzgyMDMzMjgtNDliNWFhZDctOGM3Ni00MzIwLTgyNTgtMzgwNDk2Mzk5NGU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIwVDE1NTQyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM0MjVkZmYzNTFlN2Y3YWFmZTQyYTYwNzgyMGZhNjU2Y2U4ODMxMmZmYjg0ODA0M2NiZmY2YzE0Mzc2ZTk0MzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qJYmKoeExSK90rI2Y4ncURbIocG21speiV6q9T7vu5c)
> 
> its improving

yes sir with your guidance it will be going better day by day

**Comment by HimangshuSharma01** [comment]
@sanjay-kv  Sir I was asked to remove the levels from the open source contribution as they are one line change please look into that

**Reply by sanjay-kv** [reply]
i have changed it.

**Reply by HimangshuSharma01** [reply]
> i have changed it.

Okay sir

**Comment by HimangshuSharma01** [comment]
@sanjay-kv Sir please update the link provided in the about section it redirects to the older page

https://github.com/user-attachments/assets/972076d5-c34f-4238-bfda-9dbffeb5dacb
